controllers/mysqlcluster_controller.go

```diff
@@ -260,6 +296,101 @@ func generateRandomBytes(n int) ([]byte, error) {
 	return ret, nil
 }
 
+func mergeMycnf(cnf1, cnf2 map[string]string) {
+	for k, v := range cnf2 {
+		cnf1[normalizeConfKey(k)] = v
+	}
+}
+
+func normalizeConfKey(k string) string {
+	return strings.ReplaceAll(k, "-", "_")
+}
+
+func (r *MySQLClusterReconciler) createOrUpdateConfigMap(ctx context.Context, log logr.Logger, cluster *mocov1alpha1.MySQLCluster) error {
+	cm := &corev1.ConfigMap{}
+	cm.SetNamespace(cluster.Namespace)
+	cm.SetName(cluster.Name)
+
+	op, err := ctrl.CreateOrUpdate(ctx, r.Client, cm, func() error {
+		cm.Labels = map[string]string{
+			appNameKey: fmt.Sprintf("%s-%s", appName, cluster.Name),
```

Comment:
The label `app.kubernetes.io/name` describes both application and cluster name since argocd also uses label `app.kubernetes.io/instance`. So we added an application name to it.

Reply:
@ysksuzuki 
So, what about `app.kubernetes.io/part-of` or `app.kubernetes.io/managed-by`?
https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels

Reply:
Also I think `app.kubernetes.io/instance: moco` is ignored by Argo CD.
Argo CD cares about the label only when the label value is `argocd`, doesn't it?